File: fix_layouts.py
import os

files = [
  'AdminDashboard.jsx',
  'LecturerDashboard.jsx',
  'StudentDashboard.jsx',
  'CoursesManagement.jsx',
  'StudentManagement.jsx',
  'LecturerManagement.jsx',
  'AssignmentsManagement.jsx',
  'AttendanceManagement.jsx',
  'GradesManagement.jsx',
  'LearningMaterialsManagement.jsx',
  'ReportsManagement.jsx',
  'SettingsManagement.jsx'
]

for file in files:
    filepath = os.path.join('src', 'pages', file)
    if os.path.exists(filepath):
        with open(filepath, 'r') as f:
            content = f.read()
        
        import re
        content = re.sub(r"import DashboardLayout from '\.\./layouts/DashboardLayout';\n?", "", content)
        content = re.sub(r"<DashboardLayout[^>]*>\s*", "", content)
        content = re.sub(r"\s*</DashboardLayout>", "", content)
        
        with open(filepath, 'w') as f:
            f.write(content)
        print(f"Fixed {file}")

File: fix-layouts.js
const fs = require('fs');
const path = require('path');

const files = [
  'AdminDashboard.jsx',
  'LecturerDashboard.jsx',
  'StudentDashboard.jsx',
  'CoursesManagement.jsx',
  'StudentManagement.jsx',
  'LecturerManagement.jsx',
  'AssignmentsManagement.jsx',
  'AttendanceManagement.jsx',
  'GradesManagement.jsx',
  'LearningMaterialsManagement.jsx',
  'ReportsManagement.jsx',
  'SettingsManagement.jsx'
];

files.forEach(file => {
  const filePath = path.join(__dirname, 'src', 'pages', file);
  if (fs.existsSync(filePath)) {
    let content = fs.readFileSync(filePath, 'utf8');
    
    // Remove import
    content = content.replace(/import DashboardLayout from '..\/layouts\/DashboardLayout';\n?/g, '');
    
    // Remove <DashboardLayout ...>
    content = content.replace(/<DashboardLayout[^>]*>\s*/g, '');
    
    // Remove </DashboardLayout>
    content = content.replace(/\s*<\/DashboardLayout>/g, '');
    
    fs.writeFileSync(filePath, content);
    console.log(`Fixed ${file}`);
  }
});
